src/shims/unix/fd.rs

```diff
@@ -218,6 +226,8 @@ impl FileDescriptor {
 #[derive(Debug)]
 pub struct FdTable {
     pub fds: BTreeMap<i32, FileDescriptor>,
+    /// Unique identifier for file description, used to differentiate between various file description.
+    next_file_description_id: usize,
```

Comment:
This could have been kept in the other PR, as it is not necessary on master, and fairly simple to add after the other changes of this PR